### ISSUE TITLE
Endpoint Header Options

### ISSUE THREAD

**Issue by zanettin** [ermanc/ChromeiQL#4]
Great Tool! It would be nice if user can also define custom headers to access the endpoint like authorization header.

**Comment by holm0563** [comment]
This is what i need as well

**Comment by doubleppereira** [comment]
I was trying to use https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj to inject my custom headers but without luck. 
if we have a way to inject them trought this tool would be amazing.
I need this as well.
Thanks

**Comment by jrichard0725** [comment]
Definitely would love this feature

**Comment by fnaum** [comment]
same here, we need to make it work with github enterprise API and we need to send the user and authorization token.

**Comment by rjturek** [comment]
Auth header would be so fine for GitHub Enterprise API.

**Comment by ermanc** [comment]
I agree with you all and this is on top of my todo list right now for the extension.

**Comment by martin-walsh** [comment]
+1 Auth header would be great
